core/src/test/java/org/apache/iceberg/jdbc/TestJdbcCatalog.java

```diff
@@ -234,6 +236,43 @@ public void testReplaceTxnBuilder() {
     assertThat(table.properties()).containsEntry("key1", "value1").containsEntry("key2", "value2");
   }
 
+  @Test
+  public void testReplaceTxnBuilderDefaultFormatVersion() {
```

Comment:
Same nit about extracting out common code since we will probably have 3 of these tests in the near future.